src/test.jest.testRunner.ts

```diff
@@ -125,9 +125,9 @@ async function askRunner(
 
   const askCode = parseAskScript(source);
   const askCodeSource = askCodeToSource(askCode);
-  const askCodeSource2 = askCodeToSource(
```

Comment:
These lines, if unused, should just be deleted

Reply:
@YonatanKra I agree, do you know an automated tool that we could integrate with our CI to enforce it?

Reply:
@mhagmajer eslint should be fine (`no-unused-...` rules)

Reply:
FYI, I created an issue for a linter.
https://github.com/xFAANG/askql/issues/165